datagrepper/app.py

```diff
@@ -279,11 +280,9 @@ def raw():
     # Perform our complicated datetime logic
     start = flask.request.args.get("start", None)
     end = flask.request.args.get("end", None)
-    default_delta = app.config["DEFAULT_QUERY_DELTA"]
     delta = flask.request.args.get("delta")
-    if not (start or end or delta):
-        delta = default_delta
-    start, end, delta = assemble_timerange(start, end, delta)
+    with app.app_context():
```

Comment:
This should be removed since the context is not created and destroyed in the unit tests.

Reply:
Done in 6c66902